controllers/vspherecluster_reconciler_test.go

```diff
@@ -334,25 +334,11 @@ var _ = Describe("VIM based VSphere ClusterReconciler", func() {
 			By("deleting the vspherecluster which has the secret with legacy finalizer")
 			Expect(testEnv.Delete(ctx, instance)).To(Succeed())
 
-			By("checking that the secret is deleted")
```

Comment:
What is the context for this one?

Reply:
this was unintentional for this commit, i ll revert